firebase-config/src/main/java/com/google/firebase/remoteconfig/ConfigUpdateListener.java

```diff
@@ -0,0 +1,41 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//
+// You may obtain a copy of the License at
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.remoteconfig;
+
+import androidx.annotation.NonNull;
+import javax.annotation.Nonnull;
+
+/**
+ * Event Listener for Realtime config update callbacks.
+ *
+ * @author Quan Pham
```

Comment:
Question for platform folks: Do we typically include `@author`?

Reply:
Good point! we don't use `@author`

Reply:
Removed